site/content/docs/5.3/getting-started/webpack.md

```diff
@@ -86,6 +86,7 @@ With dependencies installed and our project folder ready for us to start coding,
 
 1. **Open `webpack.config.js` in your editor.** Since it's blank, we'll need to add some boilerplate config to it so we can start our server. This part of the config tells Webpack where to look for our project's JavaScript, where to output the compiled code to (`dist`), and how the development server should behave (pulling from the `dist` folder with hot reload).
 
+   <!-- eslint-skip -->
```

Comment:
I don't know why I have to add this one here. It's not needed upstream, but I can't find the reason. If you have any idea @louismaximepiton

Reply:
I've been searching for like an hour but can't find why. I saw that we have the dependency inside Boosted imported by Storybook but I can't find why it raises an error. I've been searching inside the [eslint doc](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md) and tried to implement the various things but got the same result.

For instance, tried by tweaking all the options in `"import/no-extraneous-dependencies": ["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false, "bundledDependencies": false, "packageDir": "./"}],`